packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -1216,8 +1206,8 @@ function* watchRecovery() {
  */
 function* recoverOnPageLoad() {
   const localStorage = yield* getContext('localStorage')
-  const savedLocalStorageState: BuyAudioLocalStorageState | null = yield* call(
-    [localStorage, localStorage.getJSONValue],
+  const savedLocalStorageState = yield* call(
+    (val) => localStorage.getJSONValue<BuyAudioLocalStorageState>(val),
```

Comment:
why this rewrite?

Reply:
There's some issue with babel where you can't have `,getJSONValue<BuyAudioLocalStorage>]` without it freaking out, even tho it's totally valid TS syntax. Had to actually call the function instead of using the nice context form of `call`

Reply:
:( sad